common/arg_parser.h

```diff
@@ -141,13 +141,26 @@ class BitSet {
       bits_[i] = 0;
     }
   }
+  void fill(size_t bit = SIZE) {
```

Comment:
This should be a static function that returns a bitset. It also needs a comment to explain what it does.